src/components/Sidebar.vue

```diff
@@ -12,16 +13,18 @@ onMounted(() => spacesStore.fetch());
     <router-link :to="{ name: 'home' }" class="h-[72px] block">
       <IH-stop class="inline-block my-4 w-[32px] h-[32px] text-skin-link" />
     </router-link>
-    <UiLoading v-if="!spacesStore.loaded" class="block py-2" />
-    <div v-else class="space-y-3 p-2">
-      <router-link
-        v-for="(space, i) in spacesStore.spaces.slice(0, 5)"
-        :key="i"
-        :to="{ name: 'overview', params: { id: `${space.network}:${space.id}` } }"
-        class="block"
-      >
-        <Stamp :id="space.id" :size="32" class="!rounded-[4px]" />
-      </router-link>
-    </div>
+    <UiLoading v-if="!spacesStore.loaded" class="py-2 block" />
+    <draggable
+      v-else
+      v-model="spacesStore.starredSpacesIds"
+      :item-key="i => i"
+      class="space-y-3 p-2"
+    >
+      <template #item="{ element }">
+        <router-link :to="{ name: 'overview', params: { id: element } }" class="block">
+          <Stamp :id="element" :size="32" class="!rounded-[4px]" />
```

Comment:
We should drop networkId for Stamp.

Reply:
hm, but why these links:
[networkId+spaceId](https://stamp.fyi/avatar/gor:0x95DC6f73301356c9909921e21b735601C42fc1a8?s=64)
[spaceId](https://stamp.fyi/avatar/0x95DC6f73301356c9909921e21b735601C42fc1a8?s=64)
return the same image?